Core/HLE/sceKernelModule.cpp

```diff
@@ -1268,7 +1268,7 @@ static Module *__KernelLoadELFFromPtr(const u8 *ptr, size_t elfSize, u32 loadAdd
 		scan = g_Config.bFuncReplacements;
 #endif
 
-		bool gotSymbols = scan && reader.LoadSymbols();
+		bool insertSymbols = scan && !reader.LoadSymbols();
```

Comment:
Maybe a oneliner comment explaining the logic here?

Reply:
I'll add one, but this has something to do with the symtab stuff which I've never really messed with.

-[Unknown]